polkadot/runtime/westend/src/impls.rs

```diff
@@ -100,6 +100,12 @@ where
 		// Do `check_out` accounting since the XCM Executor's `InitiateTeleport` doesn't support
 		// unpaid teleports.
 
+		// withdraw the asset from `who`
+		let who_origin =
+			Junction::AccountId32 { network: None, id: who.clone().into() }.into_location();
+		let _withdrawn = xcm_config::LocalAssetTransactor::withdraw_asset(&wnd, &who_origin, None)
+			.map_err(|_| pallet_xcm::Error::<Runtime>::CannotWithdraw)?;
```

Comment:
The short hands `wnd` and `roc` always require me to do a double take when reading the code. If this isn't a house style I'd prefer to have the full names `westend` and `rococo`, but up to you.

Reply:
"Westend" and "Rococo" are the names of the networks, "WND" and "ROC" are the names of their native asset, respectively.